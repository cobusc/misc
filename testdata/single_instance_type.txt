0,M1,1,1
