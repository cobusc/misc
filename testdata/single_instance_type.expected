EMPTY: M1=0; M2=0; M3=0;
FULL: M1=1; M2=0; M3=0;
MOST FILLED: M1=0,0; M2=0,0; M3=0,0;
